README.md

```diff
@@ -0,0 +1,45 @@
+# terraform-aws-cross-account-role
+A Terraform module to create an IAM role for cross-account use. This module creates the role in the destination account, but does not configure access in the source account.
```

Comment:
Can we replace `destination` with `satellite`, just to keep terminology consistent?